src/providers/DataTypes.ts

```diff
@@ -6,16 +6,21 @@ import { TimeZoneDescription } from '../types/TimeZones';
 export type ProvidedUserData = {
   user?: User;
   reloadUser: () => void;
+  bootstrapped: boolean;
```

Comment:
all provider data has a bootstrapped boolean, up to provider to flag if data is sufficient